manifests/09_service_monitor.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  creationTimestamp: "2019-09-03T14:20:41Z"
+  generation: 1
```

Comment:
I don't think we need to specify `creationTimestamp` and `generation` here, do we?